.changesets/feat_local_safelist.md

```diff
@@ -0,0 +1,18 @@
+### Allow use of a local persisted query manifest for use with offline licenses ([Issue #4587](https://github.com/apollographql/router/issues/4587))
+
+This adds support to be able to pass [Persisted Query](https://www.apollographql.com/docs/graphos/operations/persisted-queries/) manifests to be used in place of the hosted Uplink version. 
+
+An example configuration would look like:
+
+```yml
```

Comment:
```suggestion
### Support local persisted query manifests for use with offline licenses ([Issue #4587](https://github.com/apollographql/router/issues/4587))

Adds experimental support for passing [persisted query manifests](https://www.apollographql.com/docs/graphos/operations/persisted-queries/#31-generate-persisted-query-manifests) to use instead of the hosted Uplink version. 

For example:

```router.yaml
```